backend/backend/public_urls.py

```diff
@@ -37,4 +34,13 @@
     # Feature flags
     path(f"{path_prefix}/flags/", include("feature_flag.urls")),
 ]
+if settings.ADMIN_ENABLED is True:
```

Comment:
@muhammad-ali-e no need for `is True`. In python it is a redundant check.
```suggestion
if settings.ADMIN_ENABLED:
```

Reply:
updated